Automattic/wp-calypso
# Add Jetpack Social products to allow for variant selection

Added Jetpack Social Basic products to allow for variant selection

<img width="714" alt="Screenshot 2022-08-29 at 2 50 07 PM" src="https://user-images.githubusercontent.com/6594561/187168632-a016a083-8de5-414d-961a-eadee73b7a4b.png">

#### Proposed Changes

* Add upcoming Jetpack Social Basic product information to support variant selection in the UI.


#### Testing Instructions



* Testing instructions
* Head to http://calypso.localhost:3000/checkout/jetpack/jetpack_social_basic_yearly to add Jetpack Social Basic Yearly to your shopping cart
* Verify that two variants appear (yearly and monthly)
* Verify that you can switch between the variants
* Verify that you can complete the purchase using credits or the store sandbox.


#### Pre-merge Checklist
